packages/core/src/common/menu/composite-menu-node.spec.ts

```diff
@@ -0,0 +1,69 @@
+// *****************************************************************************
+// Copyright (C) 2024 EclipseSource and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+import * as chai from 'chai';
+import { CompositeMenuNode } from './composite-menu-node';
+import { CompoundMenuNodeRole } from './menu-types';
+
+const expect = chai.expect;
```

Comment:
minor: why not just do `import { expect } from 'chai';` like all the other tests do?